forge-gui/res/cardsfolder/p/paliano_vanguard.txt

```diff
@@ -0,0 +1,8 @@
+Name:Paliano Vanguard
+ManaCost:1 W
+Types:Creature Human Soldier
+PT:2/2
+Draft:Draft CARDNAME face up.
+Draft:As you draft a creature card, you may reveal it, note its creature types, then turn CARDNAME face down.
+S:Mode$ Continuous | Affected$ Creature.Other+NotedTypes+YouCtrl | AddPower$ 1 | AddToughness$ 1 | Description$ Other creatures you control of a type you noted for cards named Paliano Vanguard get +1/+1.
```

Comment:
hmn, I think you kind of will have to hardcode the "named" part into these scripts too
(so that when e.g. _Sakashima the Impostor_ copies them the abilities will still work :) )

Reply:
or you could try to use getCardState in the code to navigate to the original trait host

Reply:
I put them into CardProperty. I'm not sure if I love that but its more correct for now..